### PR TITLE
version 0.6.0

### PR DESCRIPTION
**Pull request by charlierudolph**
@allewun @ricmatsui @kevgo

### CODE REVIEW

File: man/man1/git-sync.1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.TH "GIT-SYNC" "1" "01/09/2015" "Git Town 0\&.5\&.0" "Git Town Manual"
+.TH "GIT-SYNC" "1" "04/02/2015" "Git Town 0\&.6\&.0" "Git Town Manual"
```
Comment (kevgo) [inline man/man1/git-sync.1]: Clever to use tomorrow here   :)

**Comment by kevgo** [comment]
:rocket:

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
 # Git Town Release Notes
 
+## 0.6.0 (2015-04-02)
+
+* support for working without a remote repository for **git extract**, **git hack**, **git kill**, **git ship**, and **git sync**
```
Comment (allewun) [inline RELEASE_NOTES.md]: let's credit @ricmatsui for this!

Comment (kevgo) [inline RELEASE_NOTES.md]: +1

Comment (charlierudolph) [inline RELEASE_NOTES.md]: done

**Comment by charlierudolph** [comment]
@allewun @kevgo all comments addressed

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+  * implemented by our newest core committer, @ricmatsui
```
Comment (kevgo) [inline RELEASE_NOTES.md]: I'd remove the comma here

**Comment by kevgo** [comment]
Good to go once the final comment is addressed

File: RELEASE_NOTES.md (hunk 3 above)
Comment (charlierudolph) [inline RELEASE_NOTES.md]: You don't like the pause for dramatic effect? haha

File: RELEASE_NOTES.md (hunk 2 above)
Comment (ricmatsui) [inline RELEASE_NOTES.md]: Thanks! :)

**Pull request closed without merging**